src/encoded/static/components/app.js

```diff
@@ -1661,9 +1661,19 @@ const ContentRenderer = React.memo(function ContentRenderer(props) {
         // error catching
         content = <ErrorPage currRoute={routeLeaf} status={status} />;
     } else if (context) {
+        // Hack for associating the /browse/ searches with BrowseView.
+        // Otherwise it will use FileSearchView since Regisry.lookup checks for first matching view 
+        let lookupContext = context;
+        const browseIdx = context?.['@type']?.indexOf('Browse') ||  -1;
```

Comment:
VS Code is flagging one of the spaces before ```-1``` as a special character. Not sure why, but looks like a double space anyways.
```suggestion
        const browseIdx = context?.['@type']?.indexOf('Browse') || -1;
```